skyportal/app_server.py

```diff
@@ -86,6 +87,35 @@
 
 log = make_log('app_server')
 
+
+class CustomApplication(tornado.web.Application):
+    def log_request(self, handler):
+        """Writes a completed HTTP request to the logs.
+        By default writes to the python root logger.  To change
```

Comment:
What is "the python root logger"?

Reply:
this is just copy-pasted from the tornado source code, with minor changes